third_party/src/default/assets/js/main.js

```diff
@@ -419,8 +419,8 @@ function renderSimpleHTMLRecursive(obj, package, spacing) {
     var shownPackages = [];
     for (var _i = 0, _a = Object.keys(obj); _i < _a.length; _i++) {
         var key = _a[_i];
-        if (typeof obj[key] == 'object') {
-            html += renderHTMLRecursive(obj[key], package + '_' + key, spacing + '&emsp;');
+        if (typeof obj[key] === 'object') {
+            html += renderSimpleHTMLRecursive(obj[key], package + '_' + key, spacing + '&emsp;');
```

Comment:
Unrelated to this PR, we could probably delete this file and use the new Cloud Build task to generate this file in the packaged theme. This is redundant for pure source code.